src/popup/utils/utils.ts

```diff
@@ -663,6 +682,37 @@ export function isTxOfASupportedType(encodedTx: string, isTxBase64 = false) {
   return SUPPORTED_TX_TYPES.includes(SCHEMA.OBJECT_ID_TX_TYPE[txObject.tag]);
 }
 
+export function isTxDex(tx: ITx, dexContracts: IDexContracts) {
+  const { wae = [], router = [] } = dexContracts;
+
+  return !!(
+    tx.contractId
+    && tx.function
+    && (
+      Object.values(FUNCTION_TYPE_DEX).flat()
+        .includes(tx.function as TxFunctionRaw)
+    )
+    && [...wae, ...router].includes(tx.contractId)
+  );
+}
+
+export const getTxOwnerAddress = (innerTx?: ITx) => innerTx?.accountId || innerTx?.callerId;
+
+export const getOwnershipStatus = (
```

Comment:
I suggest to use regular function instead of arrow to maintain consistency within this file.